.github/workflows/b-binary-build-and-e2e-tests.yaml

```diff
@@ -205,7 +205,7 @@ jobs:
       - uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # ratchet:actions/download-artifact@v3.0.2
         id: download-artifact
         with:
-          name: kubescape-ubuntu-latest
```

Comment:
Why is the path changing?

Reply:
because the path has changed with goreleaser https://github.com/kubescape/kubescape/actions/runs/7593521407/job/20684276287